### ISSUE TITLE
proguard-rules NullPointerException  

### ISSUE THREAD

**Issue by pengyuan1990** [Instabug/android-sample#51]
Installation code ：
  new Instabug.Builder(this, "6107ea801eefd3945********")
                .setInvocationEvent(InstabugInvocationEvent.NONE)
                .build();

sdk version：4.2.6

 W/System.err: java.lang.reflect.InvocationTargetException
 W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
 W/System.err:     at com.instabug.library.h.b.a(SourceFile:82)
 W/System.err:     at com.instabug.library.Instabug$Builder.build(SourceFile:2334)
 W/System.err:     at com.instabug.library.Instabug$Builder.build(SourceFile:2358)
 W/System.err:     at com.a9bhuv.z35kb.android.XApplication.d(SourceFile:88)
 W/System.err:     at com.a9bhuv.z35kb.android.XApplication.onCreate(SourceFile:57)
 W/System.err:     at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1028)
 W/System.err:     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5658)
 W/System.err:     at android.app.ActivityThread.-wrap2(ActivityThread.java)
 W/System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1637)
 W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:105)
 W/System.err:     at android.os.Looper.loop(Looper.java:156)
 W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6523)
 W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
 W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:941)
 W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:831)
 W/System.err: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.instabug.survey.b.a.a(boolean)' on a null object reference
 W/System.err:     at com.instabug.survey.b.c.a(SourceFile:8)
 W/System.err:     at com.instabug.survey.InstabugSurvey.setSurveysAutoShowing(SourceFile:87)

**Comment by tarek360** [comment]
Hi @pengyuan1990 
Thank you for reporting this issue, it's fixed in a snapshot and it will be released in the next release.
```gradle
dependencies {
 compile 'com.instabug.library:instabug:4.2.6.5-SNAPSHOT'
}
```

add this to your app-level build.gradle file to grab snapshots: 
```gradle
repositories { 
    maven { 
        url "https://oss.sonatype.org/content/repositories/snapshots" 
    } 
}
```

**Comment by vezikon** [comment]
Hello @pengyuan1990 
This fix is now on SDK version 4.2.7
